src/Program.cs

```diff
@@ -161,6 +166,21 @@ private static IWebHost BuildHost()
                     }
                 });
 
+            string serviceName = "Ngsa.Application";
+            string serviceVersion = "1.1.0";
```

Comment:
Should this be static?

Reply:
actually i forgot i should pull this in from assembly file instead. will fix